VideoApp/VideoApp/TwilioVideo/LocalParticipant/LocalParticipantManager.swift

```diff
@@ -94,6 +93,11 @@ class LocalParticipantManager: NSObject, ObservableObject {
     func configure(identity: String) {
         self.identity = identity
     }
+    
+    func setHold(isOnHold: Bool) {
+        micTrack?.isEnabled = !isOnHold
+        cameraTrack?.isEnabled = !isOnHold
+    }
```

Comment:
@bobiechen-twilio @piyushtank I'm curious about your thoughts on this. I think this may need to be more complex to handle some edge cases. For example what happens when a user changes mute status from inside the app when the video call is on hold. It could enable the track again if not careful.

Reply:
Would it make the logic more solid if we do all in-call (room?) operations by submitting requests to CallKit? So instead of directly setting the SDK properties, we use CallKit requests (if the operation is done via app UI actions) that way UI status are in-sync both the app and CallKit.

Reply:
Yeah that is basically how I am handling mic mute at least. See CallManager.setMute. The UI calls this. 

However hold adds another layer of complexity since I now need to be concerned about what has the user done with mute AND hold. The track can get into an incorrect state if not careful. 

This is kind of complex to discuss. I may end up keeping it simple for this PR and creating a new ticket to optimize more when you and @piyushtank are back.

Reply:
I created a separate ticket to think about this edge case more.